src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -714,13 +771,13 @@ class device {
             pt_dma.put( "c2h", unitConvert(devstat.c2h[i]) );
             sensor_tree::add_child( std::string("board.pcie_dma.transfer_metrics.chan." + std::to_string(i)), pt_dma );
         }
+        
         getMemTopology( devstat );
-        // stream
 
         // xclbin
-        std::string errmsg, xclbinid;
-        pcidev::get_dev(m_idx)->sysfs_get("", "xclbinuuid", errmsg, xclbinid);
-        if(errmsg.empty()) {
+        {
```

Comment:
why remove the check for errmsg?

Reply:
No sysfs get in this function is checking for errmsg. So I thought it would be good to have some consistency.